### PR TITLE
Fix warnings in AIBL converter

### PR DESCRIPTION
**Pull request by ghisvail**

### CODE REVIEW

File: clinica/iotools/converters/aibl_to_bids/aibl_utils.py [hunk 1]
```diff
@@ -36,58 +36,65 @@ def find_T1_folder(subdirectory, path_to_T1_1):
...
+    path_to_convert = {
+        "MPRAGESAGISOp2ND",
+        "MPRAGE_SAG_ISO_p2_ND",
+        "MPRAGE_SAG_ISO_p2",
+    }
+
+    path = None
+
     for j in path_to_convert:
-        path = []
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/aibl_to_bids/aibl_utils.py:66]: This pattern is a copy-paste from `find_T1_folder`. Could be worth factorizing it out. WDYT?

File: clinica/iotools/converters/aibl_to_bids/aibl_utils.py [hunk 2]
```diff
@@ -328,12 +331,12 @@ def find_path_to_T1_ADNI(file_mri, subjects_ID, path_to_dataset):
...
     images which are ADNI compliant (as explained in the AIBL website).
-    This images differ from the others T1 of the dataset since in the
-    cvs_file is reported the exame date.
+    These images differ from the others T1 of the dataset since in the
+    cvs_file is reported the exam date.
 
     :param file_mri: in the clinical data there are two files which
     describe the  parameters of the T1 images (MRI 1.5 T and MRI 3T)
-    :param subjects_ID: subjects_id in the dataset dowloaded
+    :param subjects_ID: subjects_id in the dataset downloaded
```
Comment (NicolasGensollen) [inline clinica/iotools/converters/aibl_to_bids/aibl_utils.py:339]:
```suggestion
    :param subjects_ID: subjects_id in the downloaded dataset
```

**Review by NicolasGensollen — COMMENTED** [review]
Thanks @ghisvail !
The changes LGTM! I just have a few suggestions. Feel free to take or leave.

File: clinica/iotools/converters/aibl_to_bids/aibl_utils.py [hunk 3]
```diff
@@ -328,12 +331,12 @@ def find_path_to_T1_ADNI(file_mri, subjects_ID, path_to_dataset):
 
     This method creates a Dataframe which contains all the paths to the T1
     images which are ADNI compliant (as explained in the AIBL website).
-    This images differ from the others T1 of the dataset since in the
-    cvs_file is reported the exame date.
+    These images differ from the others T1 of the dataset since in the
+    cvs_file is reported the exam date.
```
[the remark this answers is not in the document]
Comment (ghisvail) [reply clinica/iotools/converters/aibl_to_bids/aibl_utils.py:347]: A suggestion would have been welcome here.

**Comment by ghisvail** [comment]
> Thanks @ghisvail ! The changes LGTM! I just have a few suggestions. Feel free to take or leave.

They are good suggestions. I have addressed all of them.

**Review by NicolasGensollen — APPROVED** [review]
Great! Thanks for addressing my points @ghisvail !

**Pull request merged**